<launch>
    <node pkg="stereo_click" type="mono_converter.py" name="mono_converter_node" output="screen">
		<param name="input" value="processing_points" />
        <param name="table_height" value="0.872" />
		<param name="output" value="stereo_points_3d" />
		<param name="output_frame" value="base_footprint" />
	</node>
	
	<node pkg="stereo_click" type="stereo_converter.py" name="stereo_converter_node" output="screen">
		<param name="input1" value="left_processing_points" />
		<param name="input2" value="right_processing_points" />
		<param name="output" value="stereo_points_3d" />
		<param name="output_frame" value="base_footprint" />
	</node>

    <node pkg="image_processor" type="shape_fitter_node.py" name="shirt_fitter_node" output="screen" >
        <param name="mono_converter" value="mono_converter_node" />
        <param name="stereo_converter" value="stereo_converter_node" />
        <param name="threshold" value="120" />
        <param name="model" value="shirt" />
        <param name="mode" value="shirt" />
    </node>
    
 
    <node pkg="image_view" type="image_view" name="shirt_viewer">
        <remap from="image" to="shirt_fitter_node/annotated" />
    </node>
</launch>
